### PR TITLE
test: add test case for email and password input validation

### PR DESCRIPTION
**Pull request by Ramsonodoi**

### CODE REVIEW

File: src/app/components/login/login.component.spec.ts [hunk 1]
```diff
@@ -27,6 +27,11 @@ describe('LoginComponent', () => {
     expect(component).toBeTruthy()
   })
 
+  it('should validate email format', () => {
```
Comment (Bilal-Abubakari) [inline src/app/components/login/login.component.spec.ts:30]: This does not validate email format

**Review by Bilal-Abubakari — COMMENTED** [review]
Correct your PR title as well

File: src/app/components/login/login.component.spec.ts (hunk 1 above)
Comment (Ramsonodoi) [reply src/app/components/login/login.component.spec.ts:30]: Okay thank you will do the necessary changes and push the code again

Comment (Ramsonodoi) [reply src/app/components/login/login.component.spec.ts:30]: Please changes has been done  and pushed to branch.
Please review and give feedback @Bilal-Abubakari

**Review by Bilal-Abubakari — COMMENTED** [review]
This is not a comprehensive unit test for a login component. You are only testing the validity of the email field

**Comment by Ramsonodoi** [comment]
Hello @Bilal-Abubakari  Please I have added the validity of the password field

**Pull request merged**